analytic_engine/src/meta/details.rs

```diff
@@ -311,17 +337,26 @@ impl MetaUpdateLogStore for RegionWal {
     }
 
     async fn store(&self, log_entries: &[MetaUpdateLogEntry]) -> Result<()> {
-        let mut log_batch = LogWriteBatch::new(self.region_id);
         let mut payload_batch = Vec::with_capacity(log_entries.len());
 
+        // TODO(ygf11): maybe we can build payload in encode loop.
```

Comment:
Do you mean encode payloads one by one? If so I prefer to encode them in a batch like now.

Reply:
No, we still encode in a batch.

I am considering maybe we can pass `Into<Payload>` slice instead of `Payload` slice to the encode method, then the build loop can be eliminated.

current:
```rust
 pub fn encode<P: Payload>(self, payload_batch: &[P]) -> manager::Result<LogWriteBatch> {
 ... 
}
```

after:
```rust
pub fn encode<P: Payload, I:Into<Payload>>(self, payload_batch: &[I]) -> manager::Result<LogWriteBatch> {
 ... 
}
```

Reply:
Got it 👍